services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationRepository.kt

```diff
@@ -0,0 +1,17 @@
+package fi.hel.haitaton.hanke.application
+
+import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
+import org.springframework.stereotype.Repository
+
+@Repository
+interface ApplicationRepository : JpaRepository<ApplicationEntity, Long> {
```

Comment:
A CrudRepository may be sufficient here as this does not seem to use any JpaRepository-specific functionalities.

Reply:
JpaRepository is used everywhere else in the service. Is there a benefit in using CrudRepository instead? We're not going to change the underlying database anytime soon.

Reply:
Just that there is no need for jpa repostory if you are not using it (overhead). But if it is a general practice to use jparepository then it is ok.